databallpy/data_parsers/event_data_parsers/statsbomb_parser.py

```diff
@@ -0,0 +1,507 @@
+import json
+import os
+
+import numpy as np
+import pandas as pd
+
+from databallpy.data_parsers.metadata import Metadata
+from databallpy.events import DribbleEvent, PassEvent, ShotEvent
+from databallpy.utils.constants import MISSING_INT
+from databallpy.utils.logging import create_logger
+
+LOGGER = create_logger(__name__)
+
+def load_statsbomb_event_data(
+    events_loc: str, match_loc: str, lineup_loc: str,  pitch_dimensions: tuple = (120.0, 80.0)
+) -> tuple[pd.DataFrame, Metadata, dict]:
+    """This function retrieves the metadata and event data of a specific match. The x
+    and y coordinates provided have been scaled to the dimensions of the pitch, with
+    (0, 0) being the center. Additionally, the coordinates have been standardized so
+    that the home team is represented as playing from left to right for the entire
+    match, and the away team is represented as playing from right to left.
+
+    Args:
+        events_loc (str): location of the event.json file.
+        match_loc (str): location of the match.json file.
+        lineup_loc (str): location of the lineup.json file.
+        pitch_dimensions (tuple, optional): the length and width of the pitch in meters
```

Comment:
It's probably yards. I think we need to make the conversion to meters given the other parsers. Otherwise we could get different results calculating physiological features? 1 yard = 0,9144 meters

Reply:
Makes sense, could you also build in some kind of sanity check? For instance that the length of the pitch will bee between 95 and 115 and the width between 61 and 75, just to be sure. It could be that statsbomb uses different conventions over different leagues, just raise an error if it is outside of the range, than we can easily fix it if someone comes up with an issue